source/style-guide/markup/directives/tables.txt

```diff
@@ -0,0 +1,13 @@
+================
+Table Directives
+================
+
+Use tables to format related information.
+
+In Snooty, you can only use the ``list-table`` directive to build tables.
+The ``list-table`` directive generates a formatted table from a two-level
+bullet list.
+
+Learn more about the ``list-table`` directive frmo the
```

Comment:
**Style:** See [Link and Cross-Reference Text](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#link-and-cross-reference-text).

```suggestion
To learn more about the ``list-table`` directive, see the
```

Reply:
I have issues with this style guide item, but happy to make this change for now.